deployment/common/common.yaml

```diff
@@ -21,4 +21,77 @@ spec:
   scope: Namespaced
   names:
     plural: routes
-    kind: Route
\ No newline at end of file
+    kind: Route
+  validation:
+    # openAPIV3Schema is the schema for validating custom objects.
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+            - host
+          properties:
+            strategy:
+              type: string
+              enum:
+                - RoundRobin
+                - WeightedLeastRequest
+                - Random
+            lbHealthCheck:
+              type: object
+              properties:
+                path:
```

Comment:
Should the `path` be required for the health check?

Reply:
Problem is health checks aren't required, so if I make it required, you'll have to fill it in each time.

Reply:
Aaah ok. I assumed that it would make it required for the object, but not the entire CRD.

Reply:
no wait, actually you're right, I just tested again!